crates/turbopack-css/src/process.rs

```diff
@@ -699,17 +699,18 @@ impl lightningcss::visitor::Visitor<'_> for CssModuleValidator {
         &mut self,
         selector: &mut lightningcss::selector::Selector<'_>,
     ) -> Result<(), Self::Error> {
-        if selector.iter().all(|component| !match component {
-            parcel_selectors::parser::Component::ID(_)
-            | parcel_selectors::parser::Component::Class(_) => true,
-            parcel_selectors::parser::Component::LocalName(local) => {
-                !matches!(&*local.name.0, "html" | "body")
-            }
-            _ => false,
-        }) {
+        if selector
+            .iter_raw_parse_order_from(0)
+            .all(|component| match component {
```

Comment:
I think the condition isn't correct. It should check if it contains a class name or id somewhere in the selector. I feel it's not longer doing that